### PR TITLE
fix: input field parameters

### PR DESCRIPTION
**Pull request by misamu**
Added pattern attribute to InputField.
Pattern to define allowed characters for the field

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation within the code base (if appropriate).

## Further comments

Related to [Kratos pull request 3143](https://github.com/ory/kratos/pull/3143)
- Adds `Pattern="[0-9]+"` to show error border on code field when non numeric character given
- Show error border in InputField when code field has focus and `:invalid `

### CODE REVIEW

**Comment by Benehiko** [comment]
I would like to get this merged, but the PR in Kratos has not been merged yet. https://github.com/ory/kratos/pull/3143

**Review by hperl — APPROVED** [review]
LGTM.

@Benehiko, https://github.com/ory/kratos/pull/3143 -> https://github.com/ory/kratos/pull/3561 is now merged

File: src/react-components/ory/helpers/node.tsx [hunk 1]
```diff
@@ -355,6 +355,8 @@ export const Node = ({
             defaultValue={attrs.value as string | number | string[]}
             required={attrs.required}
             disabled={attrs.disabled}
+            pattern={attrs.pattern}
+            fullWidth
```
Comment (Benehiko) [inline src/react-components/ory/helpers/node.tsx:359]: I don't believe `fullWidth` exists anymore on the inputfield.

Comment (Benehiko) [reply src/react-components/ory/helpers/node.tsx:359]: https://github.com/ory/elements/blob/main/src/react-components/input-field.tsx#L14-L22C2

**Review by Benehiko — CHANGES_REQUESTED** [review]

**Review by Benehiko — APPROVED** [review]

**Pull request merged**